packages/react-components/react-accordion/library/src/components/AccordionHeader/useAccordionHeader.tsx

```diff
@@ -83,7 +84,14 @@ export const useAccordionHeader_unstable = (
     expandIcon: slot.optional(expandIcon, {
       renderByDefault: true,
       defaultProps: {
-        children: <ChevronRightRegular style={{ transform: `rotate(${expandIconRotation}deg)` }} />,
+        children: (
+          <ChevronRightRegular
```

Comment:
I plan to implement [the same logic for the icon](https://github.com/microsoft/fluentui/compare/master...akatrukhin:feat/accordion-icon-animation?expand=1). However, I have concerns that it might overcomplicate the implementation and potentially introduce accessibility issues with allowing the custom motion update